src/components/governance/PageProposals.vue

```diff
@@ -35,7 +35,9 @@
           @click.native="onPropose"
         />
       </div>
+      <TmDataLoading v-if="$apollo.loading" />
```

Comment:
why not use the TmPage?

Reply:
I wanted the loading indicator underneath the 'Create Proposal' button. i.e. Allow the UI to be as functional as fast as possible.

Reply:
I get it! If this is the behavior we want, could be worth standardizing the page layout around it. putting the action buttons into one section and the data into another section.